### ISSUE TITLE
Local quickstart documentation for CLI

### ISSUE THREAD

**Issue by khaale** [uber/cadence#2168]
https://github.com/uber/cadence/blob/master/docker/README.md contains instruction for CLI usage
```
For example to register new domain 'test-domain' with 1 retention day
`docker run --network=host --rm ubercadence/cli:master --do test-domain domain register -rd 1`
```
which fails with the following message
```
Error: Option global_domain must be provided.
```

I suppose that it worth adding `--global_domain false` to documentation.

**Comment by khaale** [comment]
Could be fixed by #2169

**Comment by dmetzgar** [comment]
Fixed by #2172
